apps/vaults/components/details/tabs/VaultDetailsStrategies.tsx

```diff
@@ -0,0 +1,177 @@
+								</b>
+							</div>
+
+							<div className={'col-span-2 flex flex-col space-y-2 bg-neutral-200 p-2 md:p-4'}>
+								<p className={'text-base text-neutral-600'}>{'Total Gain'}</p>
+								<b className={'font-number text-lg text-neutral-900'}>
+									{`${formatAmount(formatToNormalizedValue(
+										formatBN(strategy?.details?.totalGain).sub(formatBN(strategy?.details?.totalLoss)),
+										currentVault?.decimals
+									), 0, 0)} ${currentVault.token.symbol}`}
+								</b>
+							</div>
+						</div>
+
+						<div className={'flex flex-col space-y-4 bg-neutral-200 p-2 md:p-4'}>
+							<p className={'text-base text-neutral-600'}>{'Risk score'}</p>
+							<div className={'mt-0 grid grid-cols-1 gap-x-12 gap-y-2 md:grid-cols-2'}>
+								<div className={'flex flex-row items-center justify-between'}>
+									<p className={'text-sm text-neutral-500'}>{'TVL Impact'}</p>
+									<p className={'font-number text-sm text-neutral-900'}>{strategy?.risk?.riskDetails?.TVLImpact}</p>
+								</div>
+								<div className={'flex flex-row items-center justify-between'}>
+									<p className={'text-sm text-neutral-500'}>{'Audit Score'}</p>
+									<p className={'font-number text-sm text-neutral-900'}>{strategy?.risk?.riskDetails?.auditScore}</p>
+								</div>
+								<div className={'flex flex-row items-center justify-between'}>
+									<p className={'text-sm text-neutral-500'}>{'Code Review Score'}</p>
+									<p className={'font-number text-sm text-neutral-900'}>{strategy?.risk?.riskDetails?.codeReviewScore}</p>
+								</div>
+								<div className={'flex flex-row items-center justify-between'}>
+									<p className={'text-sm text-neutral-500'}>{'Complexity Score'}</p>
+									<p className={'font-number text-sm text-neutral-900'}>{strategy?.risk?.riskDetails?.complexityScore}</p>
+								</div>
+								<div className={'flex flex-row items-center justify-between'}>
+									<p className={'text-sm text-neutral-500'}>{'Longevity Impact'}</p>
+									<p className={'font-number text-sm text-neutral-900'}>{strategy?.risk?.riskDetails?.longevityImpact}</p>
+								</div>
+								<div className={'flex flex-row items-center justify-between'}>
+									<p className={'text-sm text-neutral-500'}>{'Protocol Safety Score'}</p>
+									<p className={'font-number text-sm text-neutral-900'}>{strategy?.risk?.riskDetails?.protocolSafetyScore}</p>
+								</div>
+								<div className={'flex flex-row items-center justify-between'}>
+									<p className={'text-sm text-neutral-500'}>{'Team Knowledge Score'}</p>
+									<p className={'font-number text-sm text-neutral-900'}>{strategy?.risk?.riskDetails?.teamKnowledgeScore}</p>
+								</div>
+								<div className={'flex flex-row items-center justify-between'}>
+									<p className={'text-sm text-neutral-500'}>{'Testing Score'}</p>
+									<p className={'font-number text-sm text-neutral-900'}>{strategy?.risk?.riskDetails?.testingScore}</p>
+								</div>
```

Comment:
Let's replace this with a tuple [label, value] and map, it will improve readability, something like
```typescript
{[
	['TVL Impact', strategy?.risk?.riskDetails?.TVLImpact],
	['Audit Score', strategy?.risk?.riskDetails?.auditScore],
	// ...
].map(([label, value]): ReactElement => (
	<div className={'flex flex-row items-center justify-between'} key={label}>
		<p className={'text-sm text-neutral-500'}>{label}</p>
		<p className={'font-number text-sm text-neutral-900'}>{value}</p>
	</div>
))}
```

Reply:
Nice! Even better, small follow up on this https://github.com/yearn/yearn.fi/pull/13#discussion_r1056116476